src/components/application/appsco-application-sharing-settings.html

```diff
@@ -418,6 +417,11 @@
                         application: application
                     });
 
+                }.bind(this), function() {
+                    if (request.response.code === 1497962487) {
```

Comment:
There is error component called 'appsco-api-errors' which holds translations for errors based on error codes. Example of use: src/company/appsco-company-add-domain.html.
Add this error in 'appsco-api-errors' component.